integration-tests/gatsby-source-wordpress/__tests__/index.js

```diff
@@ -18,7 +19,9 @@ const isWarmCache = process.env.WARM_CACHE
 const testOnColdCacheOnly = isWarmCache ? test.skip : test
 
 describe(`[gatsby-source-wordpress] Build default options`, () => {
-  beforeAll(done => {
+  beforeAll(async done => {
+    await urling({ url: `http://localhost:8001/graphql`, retry: 15 })
```

Comment:
`urling` is cool, haven't seen this before! I've definitely written it a few times in bash 😂

Reply:
ha! I've written it a few times too. I grabbed this package to try to save a few minutes :stuck_out_tongue_winking_eye: